### PR TITLE
Match a few nonmatchings and delete unused asm

### PR DESCRIPTION
**Pull request by lepelog**

### CODE REVIEW

File: src/f/f_op/f_op_scene_iter.cpp [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+extern node_list_class lbl_803A3918;  // f_op_scene_tag::g_fopScnTg_SceneList
 
-asm void* fopScnIt_Judge(fop_ScnItFunc pFunc1, void* pFunc2) {
-    nofralloc
-#include "f/f_op/f_op_scene_iter/asm/func_8001EC74.s"
+void* fopScnIt_Judge(fop_ScnItFunc pFunc1, void* pData) {
+    struct {
+        fop_ScnItFunc mFunc;
+        void* mpData;
+    } iterParams;
```
Comment (notyourav) [inline src/f/f_op/f_op_scene_iter.cpp:17]: Is there anywhere else this is used? A bit unusual to have an inline definition like this

Comment (lepelog) [reply src/f/f_op/f_op_scene_iter.cpp:17]: I don't think so? It also doesn't matter for matching if it's an anonymous struct like this or a named one outside of the function. This is how we have it currently for all the other iter functions like this

Comment (notyourav) [reply src/f/f_op/f_op_scene_iter.cpp:17]: Gotcha, I wasn't aware of that. Perhaps that can be addressed in a different commit.

**Review by notyourav — APPROVED** [review]

File: src/f/f_op/f_op_scene_iter.cpp (hunk 1 above)
Comment (lepelog) [reply src/f/f_op/f_op_scene_iter.cpp:17]: Seems like I'm misremembering it, in f_op_actor_iter it's written like this, but the iters in f_pc have that struct named and outside of the function

**Pull request merged**